packages/@aws-cdk/aws-appmesh/test/virtual-node.test.ts

```diff
@@ -478,8 +470,51 @@ describe('virtual node', () => {
             ],
           },
         });
+      });
+
+      test('should add a backend virtual service to the resource without using a CDK Ref', () => {
+        // GIVEN
+        const stack = new cdk.Stack();
+
+        // WHEN
+        const mesh = new appmesh.Mesh(stack, 'mesh', {
+          meshName: 'test-mesh',
+        });
+
+        const node = new appmesh.VirtualNode(stack, 'test-node', {
+          mesh,
+          serviceDiscovery: appmesh.ServiceDiscovery.dns('test'),
+        });
 
+        const myVirtualService1 = new appmesh.VirtualService(stack, 'service-1', {
+          virtualServiceProvider: appmesh.VirtualServiceProvider.virtualNode(node),
+        });
 
+        const myVirtualService2 = new appmesh.VirtualService(stack, 'service-2', {
+          virtualServiceProvider: appmesh.VirtualServiceProvider.virtualNode(node),
+          virtualServiceName: 'service2.domain.local',
```

Comment:
Can we not pass `virtualServiceName` here? I don't think we have a test yet that covers that the automated Virtual Service naming works with `Backend.virtualService()`.

Reply:
That's covered by `myVirtualService1`, but I'll break them into two different tests.